File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.java
class Solution {
    public String convertToTitle(int n) {
        // StringBuilder sb = new StringBuilder();
        String s = "";
        while(n > 0){
            int curr = n % 26;
            if(curr == 0){
                curr  = 26;
            }
              s = (char)( 'A' + (curr - 1)) + s;
            n = n / 26;
            if(curr == 26){
                n--;
            }
        }
        return s;
    }
}

File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.cpp
class Solution {
public:
    string convertToTitle(int columnNumber) {
        string ans;
        while(columnNumber) {
            columnNumber--;
            ans = ans + (char)(columnNumber % 26 + 'A');
            columnNumber = (columnNumber) / 26;
        }
        reverse(ans.begin(),ans.end());
        return ans;
    }
};